website/docs/d/netapp_pool.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "NetApp"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_netapp_pool"
+sidebar_current: "docs-azurerm-datasource-netapp-pool"
+description: |-
+Gets information about an existing NetApp Pool
+---
+
+# Data Source: azurerm_netapp_pool
+
+Uses this data source to access information about an existing NetApp Pool.
+
+
+## NetApp Pool Usage
+
+```hcl
+data "azurerm_netapp_pool" "example" {
+  resource_group_name = "acctestRG"
+  account_name        = "acctestnetappaccount"
+  name                = "acctestnetapppool"
+}
+
+output "netapp_pool_id" {
+  value = "${data.azurerm_netapp_pool.example.id}"
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the NetApp Pool.
+
+* `account_name` - (Required) The name of the NetApp account.
```

Comment:
Could we make this more datasource sounding:
```suggestion
* `account_name` - (Required) The name of the NetApp account where the NetApp pool exists.
```

Reply:
Fixed